apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java

```diff
@@ -31,10 +32,12 @@
         .baseUrl(BASE_URL)
         .client(okHttpClient)
         .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
-        .addConverterFactory(new ApolloConverterFactory.Builder().build())
+        .addConverterFactory(new ApolloConverterFactory.Builder()
+            .withResponseFieldMapper(FeedQuery.Data.class, new FeedQuery.Data.Mapper(FeedQuery.Data.FACTORY))
```

Comment:
Yes user must provide Mapper instance for each query top level Data class. Converter then will use it to parse query response. So basically number of queries == number of mapper registrations